astroquery/tests/test_query.py

```diff
@@ -1,39 +1,388 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+import pytest
+import requests
+import logging
+from pathlib import Path
+from requests.models import Response
+from requests.structures import CaseInsensitiveDict
 from astroquery.query import BaseQuery, BaseVOQuery
+from astroquery.utils.mocks import MockResponse
+from itertools import product
 
+# Test data directory
+DATA_DIR = Path(__file__).parent / 'data'
+DATA_DIR.mkdir(exist_ok=True)
 
-class with_VO(BaseVOQuery, BaseQuery):
```

Comment:
Why are these deleted? I want to keep the multiple inheritances case covered as that is what we actually do in the code at a lot of places!

Reply:
there was no good reason for that.  I used AI to help me write tests, I didn't notice that it was deleting them.  I've added them back.  Thanks for noting this!